lib/python_helper_symbolic_types.py

```diff
@@ -0,0 +1,573 @@
+        },
+        'golden_ratio' : {
+            'unicode_symbol' : u'𝞍',
+            'latex_symbol'   : r'\varphi',
+            'sympy_impl'     : S.GoldenRatio,
+            'docstr'         : 'The golden ratio',
+        },
+        'catalanK'     : {
+            'unicode_symbol' : u'𝜥',
+            'latex_symbol'   : r'\kappa',
+            'sympy_impl'     : S.Catalan,
+            'docstr'         : 'Catalan\'s constant (K, or G)',
+        },
+        'euler_gamma'  : {
+            'unicode_symbol' : u'𝜸',
+            'latex_symbol'   : r'\gamma',
+            'sympy_impl'     : S.EulerGamma,
+            'docstr'         : 'The Euler-Mascheroni constant',
+        },
+        'zeta2'        : {
+            'unicode_symbol' : u'𝜻(2)',
+            'latex_symbol'   : r'\zeta(2)',
+            'sympy_impl'     : sympy.zeta(2),
+            'docstr'         : 'The famous Basel constant (PI^2/6)', 
+        },
+        'zeta3'        : {
+            'unicode_symbol' : u'𝜻(3)',
+            'latex_symbol'   : r'\zeta(3)',
+            'sympy_impl'     : sympy.zeta(3),
+            'docstr'         : 'Apery\'s irrational constant',
+        },
+        'log2'         : {
+            'unicode_symbol' : u'㏑2',
+            'latex_symbol'   : r'\ln(2)',
+            'sympy_impl'     : sympy.log(2, sympy.E),
+            'docstr'         : 'The natural logarithm of two',
+        },
+        'sqrtpi'       : {
+            'unicode_symbol' : u'√𝜋',
+            'latex_symbol'   : r'\sqrt{\pi}',
+            'sympy_impl'     : sympy.sqrt(S.Pi),
+            'docstr'         : 'The square root of PI',
+        },
+        'sqrt2'        : {
+            'unicode_symbol' : u'√2',
+            'latex_symbol'   : r'\sqrt{2}',
+            'sympy_impl'     : sympy.sqrt(2),
+            'docstr'         : 'The square root of two',
+        },
+        'log2e'        : {
+            'unicode_symbol' : u'㏒₂(e)',
+            'latex_symbol'   : r'\log_2(e)',
+            'sympy_impl'     : sympy.log(sympy.E, 2),
+            'docstr'         : 'The base-2 logarithm of E',
+        },
+        'log10e'       : {
+            'unicode_symbol' : u'㏒₁₀(e)',
+            'latex_symbol'   : r'\log_{10}(e)',
+            'sympy_impl'     : sympy.log(sympy.E, 10),
+            'docstr'         : 'The base-10 logarithm of E',
+        },
+        'ln10'         : {
+            'unicode_symbol' : u'㏑10',
+            'latex_symbol'   : r'\ln(10)',
+            'sympy_impl'     : sympy.log(10, sympy.E),
+            'docstr'         : 'The natural logarithm of ten',
+        },
+        'cbrt_omega'   : {
+            'unicode_symbol' : u'𝟂',
+            'latex_symbol'   : r'\omega',
+            'sympy_impl'     : (1 + sympy.I * sympy.sqrt(3)) / 2,
+            'docstr'         : 'The primitive cube root of unity',
+        },
+    }
+
+    OPTION_NAMED_UNITS = "named-units"
+    NAMED_UNITS_DICT = {
```

Comment:
There's a lot of commented code after this point, can you either clean it up or explain why it's there?

Reply:
@jonatanschroeder 
I think the commented code should stay for a few reasons:
* It is a perfectly valid template for future functionality that is *useful* to applications in CS and physical sciences (common units and physical constants are a good thing to have).
* It doesn't hurt to leave it in.
* If the submission process becomes more streamlined, and I actually find time to write code again, I will eventually add the freaking functionality.

Here is what is missing. The ``scipy`` library, though good and with a lot of useful stuff in it waiting to be tapped, is not by default compatible with ``sympy`` expressions. An adapter to "make it so" (as my real code boss has joked with me about for my RA job at GA Tech) is necessary. I do not have time to fiddle with all of the nuances to make it happen yet. At some point these features will be good to have.